streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java

```diff
@@ -1572,7 +1588,10 @@ public void addSuccessor(final TopologyDescription.Node successor) {
 
         @Override
         public String toString() {
-            return "Sink: " + name + " (topic: " + topic() + ")\n      <-- " + nodeNames(predecessors);
+            if (topic() != null) {
```

Comment:
For consistency: We should check `if (topicNameExtractor instanceof StaticTopicNameExtractor)` -- make the code more readable as it's the same condition as used in other method?